package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nahmii-sdk",
-  "version": "3.1.1",
+  "version": "3.1.2",
```

Comment:
I think this is OK since there are hardly any users at present. The constructor change *might* be considered a breaking change for some though!?!?

Reply:
I get your point, and I thought about it myself. The way I see it is that we are now fixing a bug in the original implementation, and thus a patch release is appropriate.